system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -940,7 +973,10 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 /// Migrations to apply on runtime upgrade.
-pub type Migrations = ();
+pub type Migrations = (
```

Comment:
something else to check:

I see Polkadot AH has [a migration for "Uniques" v0->v1](https://github.com/polkadot-fellows/runtimes/blob/97f81f9b9d759bbb5f7f64431ba71fa3718b2348/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L887) while Kusama AH doesn't (also couldn't find it ever having it in `git log -p`).

Maybe it's not needed, but better to ask.

Reply:
yes, not needed, KusamaAssetHub has already version 1:
![image](https://github.com/polkadot-fellows/runtimes/assets/8159517/02feac25-dfd7-401c-8ecf-31e8348d7416)